### PR TITLE
Add tests

### LINKED ISSUE #2: Add travis-ci tests
Refs https://discourse.algolia.com/t/do-you-have-test-credentials-that-i-could-use-for-my-opensource-plugin-to-run-tests-against-algolia/4956

### PR DESCRIPTION
**Pull request by Natim**
r? @coulix
Fixes #2

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+addons:
+  apt: 
+  postgresql: '9.5'
+env:
+  matrix:
+  - TOX_ENV=py35
+  - TOX_ENV=kinto-master
+  - TOX_ENV=flake8
+  global:
+  - secure: SZdf3p7TBW0T/MZ2ptTilt6UbEx25bxT4EXLn3aLtKAKqZh9phAUZ4QCms65g9MXXJWjbE87BHOUHStazyKgvp9TGCbOJM36hkALPYWTJFQymJpWTHGtYKGhfvJ7gRFo4Z6Ymbnip4sFwxVOlTlP+TtP4NEQFoRevci4GQz5VQj5f2x0hZtEf+14gSaZEF6JGYFvjSXn1aGDQjWvhowWgjt14aN0AWozO75bmnqRvi/MMApaJt3H8w5H/vA0AigUMQafy14bpeAN+pMDHxsdqiiwZLekmk7sHDFHbTZCv0cT7P4/0ynAxJ2T6KW62i8M0YBpFc/p/yThBVN3AquuIAAsf5RlACLoBdNQaGkopeqjU4fF4w2qS5N9xE6JqCZ3ZbhqPG1i3lTwGGmUVCJCdqL9o12H85ptpLufqSXpmBXQSDMtVhJi22sAjCZMmTFpjHeQ/qNbnLBkfFzZ/gBv8YRGAcvuBJRXxL/6g0uK+lqR9zbwZgNhgguw3nlQA9Eb9bc1XacnbQC3Je1nC8UTiJOFY2+l0hRPLeirxAlI5DxhX5PQmkuNIyohpLrx3UHUWXUDSOqe1sXj4LawOXpowqagGztuzv9CqC0dQZvQJmV9kkh4EgVT2718phS8wu5p3KZPvN07NdHe7jj/7hhMv6hSw7Wb6GcJEpV1ct+t/b4=
```
Comment (Natim) [inline .travis.yml]: This is `ALGOLIA_APPLICATION_ID`

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - secure: m1zriBApPd8l+Ym+QrSGNt9aFWNwaaRRvhs4d6VtUqqpRLBVfdYV3KEN0u9zclcsxnDOzb53QPMQSCJVZfpnI+78yPrLjYKBxnbNlu8+pT3SATD7q3rc4KAT3HjQoe9LZR8CLdhTUL7QWf3rYUIbMwit4quMq5XatbisCn1p+iUcBK4PVziCUJxiNXFD1nEfnS5AMBgGcYFtG/aerMmysn9+8wLclw+55zWZVWtQNEvv0UFXd2cwBypikhHRACQeYiAby2blxl5GS8kR/nPgXyhfDag69rtsYtfikFAbngfuXlZrYRYq/iyQMS6IajW9Vic3sEobMCcwmQW++9Qek3tcphOeahc+usStLI/lCzZ3FDPpSUu8FKbQzttZB3rfPuOqq5RphCDxgXkjbxfdHX16FschyYdDB3O2rmvBeiCWH+oIWS/Ud99QL8729/9b0NRz79qcyfgj5PcTcwIpQmGS8fDzFhBmU3qdF+JUs+YnwAYlH4q0pSs4z2RGQhzgvjshzZ62KPNsWMv1A4h4p7FQ2GsjbNhn50BqJGwMYj+gaW+MO21LleBRM4lZPbq1cGZmBP9DLLzxHPoT59CA44pWoPoaLEdkpInVeJNEfx7eNz+TtaASpNkLZI/0KWcjzZL0f7OT50XRuhirtqs6X6+32XU0S4gBqoBvdKc/Y+c=
```
Comment (Natim) [inline .travis.yml]: This is `ALGOLIA_API_KEY`

File: kinto_algolia/views.py [hunk 3]
```diff
@@ -30,19 +29,22 @@ def search_view(request, **kwargs):
     bucket_id = request.matchdict['bucket_id']
     collection_id = request.matchdict['collection_id']
 
+    # algoliasearch doesn't support pagination
+    # https://github.com/algolia/algoliasearch-client-python/issues/365
```
Comment (coulix) [inline kinto_algolia/views.py]: Is that still a thing?

Comment (Natim) [reply kinto_algolia/views.py]: Yes I created https://github.com/Kinto/kinto-algolia/issues/4 so that we can investigate

**Pull request merged**